examples/dao/scripts/deploy.js

```diff
@@ -20,13 +20,15 @@ async function run(runtimeEnv, deployer) {
 	const minDuration = 1 * 60; // 1min (minimum voting time in number of seconds)
 	const maxDuration = 5 * 60; // 5min (maximum voting time in number of seconds)
 	const url = "www.my-url.com";
+	const daoName = "DAO";
```

Comment:
Is `daoName` different from `Dao App Name`?

Reply:
yes, we use dao app name to store data in check point, while daoName is what we need to display on UI in sigma dao